content_scripts/inject_script.coffee

```diff
@@ -0,0 +1,16 @@
+injectScripts = ["pages/addEventListener_hook.js"]
+
+fetchFileContents = (extensionFileName) ->
+  req = new XMLHttpRequest()
+  req.open("GET", chrome.runtime.getURL(extensionFileName), false) # false => synchronous
+  req.send()
+  req.responseText
+
+for script in injectScripts
+  # Inject the script, which will be executed before the page scripts, as long as it is injected directly as
+  # text.
+  # TODO(mrmr1993): Find a reasonable way to inline the scripts to inject here at build time, since doing a
+  # synchronous XMLHttpRequest will slow down every page load.
+  scriptEl = document.createElement "script"
+  scriptEl.innerHTML = fetchFileContents script
+  document.documentElement.insertBefore scriptEl, document.documentElement.firstElementChild
```

Comment:
In my tests, I couldn't get a `<script src="...">` to run before page scripts in `<head>`, hence why #1405 had 2 injection methods. Feel free to change the line above to `scriptEl.src = script`; if you find it detects the clickable element for a page with the following script in `<head>` then I'm happy to change it:

```javascript
var clickable = document.createElement("div");
clickable.innerHTML = "test";
clickable.addEventListener("click", function(){alert()});
document.addEventListener("DOMContentLoaded", function(){
  document.body.appendChild(clickable);
});
```